lib/zold/remotes.rb

```diff
@@ -242,9 +244,16 @@ def if_present(host, port)
       save(list)
     end
 
+    def read_mtime
+      CSV.read(file, skip_lines: /.+,\d+,\d+,\d+/).map do |r|
+        @mtime = Time.at(r[1].to_i)
+      end
+    end
+
     def load
+      read_mtime
       @mutex.synchronize do
-        raw = CSV.read(file).map do |r|
+        raw = CSV.read(file, skip_lines: /mtime,\d+/).map do |r|
```

Comment:
@emilianodellacasa why not use the filesystem mtime?

Reply:
@ledestin It is a very good idea, implemented. Thanks a lot!